src/shared/condition.c

```diff
@@ -578,8 +583,63 @@ static int condition_test_firmware_devicetree_compatible(const char *dtcarg) {
         return strv_contains(dtcompatlist, dtcarg);
 }
 
+static int condition_test_firmware_smbios_field(const char *expression) {
+        int r;
```

Comment:
Please put `int r` at the end of the declaration list, see CODING_STYLE.

Reply:
Done